src/app/tabs/test-tab.js

```diff
@@ -30,11 +30,11 @@ module.exports = class TestTab extends ViewPlugin {
     this.appManager = appManager
     this.renderer = renderer
     this.baseurl = 'https://solc-bin.ethereum.org/bin'
-    appManager.event.on('activate', (name) => {
-      if (name === 'solidity') this.updateRunAction(fileManager.currentFile())
+    appManager.event.on('activate', (profile) => {
```

Comment:
Same as above, if you only need `name` you can deconstruct the argument if you want:
```javascript
appManager.event.on('activate', ({ name }) => {
  if (name === 'solidity') this.updateRunAction(fileManager.currentFile())
})
```